docs/_source/docs/stack_config.rst

```diff
@@ -205,6 +206,22 @@ Examples include:
 
 ``on_failure: "DELETE"``
 
+disable_rollback
+~~~~~~~~~~~~~~~~
+* Resolvable: No
+* Can be inherited from StackGroup: Yes
+* Inheritance strategy: Overrides parent if set
+
+This parameter describes the action taken by CloudFormation when a Stack fails
+to create or update, default is False. This option can be set from the stack
+config or from the Sceptre CLI commands to deploy stacks. Enabling disable-rollback
+using the CLI option disables rollback globally for all stacks. This option
+overrides on_failure since Cloudformation does not allow setting both on deployment.
+For more information and valid values see the `AWS Documentation`_.
+
+Examples include:
```

Comment:
How about just "Example:"... ? 

"Examples include" doesn't really make sense for such a simple example. It's not like there are lots of possible configurations for this.

Reply:
fixed